content/api/servo-assembly/ultrasound.md

```diff
@@ -2,17 +2,16 @@
 title: Ultrasound
 ---
 
-We noticed many teams used ultrasound sensors to assist the vision system for finer distance measurement. So we've built in the mechanisms for reading and measuring distances using ultrasound right into the kit!
+We noticed many teams used ultrasound sensors to assist the vision system for finer distance measurement, so we've built the mechanisms for reading and measuring distances using ultrasound right into the kit!
```

Comment:
Are we also providing ultrasound hardware? What happens if the competitors provide their own {additional,replacement,etc.}?

Reply:
To be honest, I think this whole paragraph feels out of place. We don't write like this for any other features of the kit such as servo control or vision, which makes it feel somewhat like another case of SR-vs-SB, something I'd like us to avoid.

I don't think we'd lose anything by removing this paragraph and replacing it with something along the lines of "The kit supports any ultrasonic sensor module compatible with the HC-SR04 standard."

Thoughts?

Reply:
(this is probably worthy of separate PR)

Reply:
Yeah, your suggestion is a much better way of presenting this; even better if we can link to the standard too.

Reply:
Broken out into #77.

Unfortunately, it doesn't seem like there is an official standard for ultrasound sensors. It just looks like the vast majority of clones of this particular type of ultrasound module are compatible, and refer to themselves as "HC-SR04".